content/en/database_monitoring/setup_postgres/troubleshooting.md

```diff
@@ -189,7 +189,15 @@ See the section on [truncated query samples](#query-samples-are-truncated) for i
 
 #### Postgres extended query protocol
 
-If a client is using the Postgres [extended query protocol][9] or prepared statements, the Datadog Agent is unable to collect explain plans due to the separation of the parsed query and raw bind parameters. If the client provides an option to force using the simple query protocol, then turning that on enables the Datadog Agent to collect execution plans.
+If a client is using the Postgres [extended query protocol][9] or prepared statements on Postgres version 12 or newer, enable the following beta feature in the [Postgres integration config][19].
```

Comment:
This explanation jumps to a solution without explaining the background. Can you rewrite this for an audience who does not have any context about parameterized queries and explain plans.